### ISSUE TITLE
Wrong update count in report and organisation overview page (estimate: 8)

### ISSUE THREAD

**Issue by Geerts** [akvo/akvo-rsr#2237]
1. In the reportserver report wrong update counts are shown because the project parameter is already set to 42. 
2. On the [organisation page](http://rsr.test.akvo.org/en/organisation/405/) "total updates" is way higher than the [update overview ](http://rsr.test.akvo.org/en/updates/?partner=405)filtered on the specific organisation.

**Comment by Geerts** [comment]
My output is:
![image](https://cloud.githubusercontent.com/assets/13657774/15862225/4e1c0fd4-2ccf-11e6-8477-79a23fe164fd.png)

Based on: 
```
SELECT
  count(DISTINCT rsr_project.id) AS project_count,
  count(DISTINCT rsr_projectupdate.id) AS update_count,
  
  rsr_country.name

FROM rsr_project
  LEFT JOIN rsr_publishingstatus ON rsr_project.id = rsr_publishingstatus.project_id
  LEFT JOIN rsr_partnership ON rsr_project.id = rsr_partnership.project_id
  LEFT JOIN rsr_projectlocation ON rsr_projectlocation.location_target_id = rsr_project.id
  LEFT JOIN rsr_country ON rsr_country.id = rsr_projectlocation.country_id
  LEFT JOIN rsr_projectupdate ON rsr_project.id = rsr_projectupdate.project_id

WHERE rsr_partnership.organisation_id = ?
      AND rsr_publishingstatus.status = 'published'

GROUP BY rsr_country.name


```

**Comment by nadiagorchakova** [comment]
@stellanl could you have a look?

**Comment by stellanl** [comment]
Same bug as #2416

**Comment by punchagan** [comment]
Closed with #2438
